libs/island-ui/core/src/lib/Checkbox/Checkbox.tsx

```diff
@@ -28,6 +28,7 @@ export interface CheckboxProps {
   /** subLabel and rightContent can only be used if the 'large' prop set to true */
   subLabel?: React.ReactNode
   rightContent?: React.ReactNode
+  children?: React.ReactNode
```

Comment:
We dropped the children idea with checkboxes, so we can clean up all island-ui changes?